src/lib/y2storage/proposal/autoinst_devices_planner.rb

```diff
@@ -188,6 +199,22 @@ def add_common_device_attrs(device, section)
         device.filesystem_type = section.type_for_filesystem
       end
 
+      # Set device attributes related to snapshots
+      #
+      # This method modifies the first argument
+      #
+      # @param device  [Planned::Device] Planned device
+      # @param drive_section [AutoinstProfile::DriveSection] AutoYaST specification
+      def add_snapshots(device, drive_section)
+        return unless device.respond_to?(:root?) && device.root?
+
+        # Always try to enable snapshots if possible
+        snapshots = true
```

Comment:
np: I would prefer a ternary operation, but I do not have an strong opinion.

Reply:
I found it to be less readable

Reply:
Or at less, less obvious